iphone/Classes/KrollBridge.m

```diff
@@ -1149,6 +1149,26 @@ - (TiModule *)loadNodeModules:(NSString *)path withDir:(NSString *)start withCon
 	return nil;
 }
 
+- (TiModule * _Nullable)cachedRequireModuleFromPath:(NSString *)path
+{
+	// See if we cached this path already
+	TiModule *module = [modules objectForKey:path];
+	if (module != nil) {
+		return module;
+	}
+
+	// Load the path if it does not exist in the path-cache so far
+	module = [self loadAsFileOrDirectory:[[path substringFromIndex:1] stringByStandardizingPath] withContext:context];
```

Comment:
You're dropping the first character here everytime, even though the places you replace with this method call isn't always do so.
I'd just take the path as-is. (or still do the stringByStandardizingPath call)
```
module = [self loadAsFileOrDirectory:[path stringByStandardizingPath] withContext:context];
```